### PR TITLE
[New] Add big-integer@1.6.26 w/ npm auto-update

### PR DESCRIPTION
**Pull request by cy91244**
Pull request for issue: #12312

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/peterolson/BigInteger.js
 * Official website (optional, not the repository):
 * NPM package url (optional): https://www.npmjs.com/package/big-integer
 * License and its reference: Unlicense
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 28
   - Count of stars: 456
   - Count of forks: 85
 * NPM download stats (optional):
   - Downloads in the last day: 16752
   - Downloads in the last week: 232540
   - Downloads in the last month: 983296

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 800 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/big-integer/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/peterolson/BigInteger.js.git"
+  },
+  "npmName": "big-integer",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "*.min.js"
```
Comment (extend1994) [inline ajax/libs/big-integer/package.json]: cdnjs need pre-minified file `BigInteger.js`
Let's use more flexible pattern to only exclude `BigInteger.d.ts`(not for front-end) & `BigInteger.test.js`(just for test).

Comment (cy91244) [reply ajax/libs/big-integer/package.json]: I have modified the pattern to be `BigInteger!(*.test.*|*.d.*)`.
Let me know if it's feasible, thanks.

Comment (extend1994) [reply ajax/libs/big-integer/package.json]: Nice @cy91244 
Just one point: `*.d.ts` could be better than `*.d.*` because typescript format postfix.

Comment (cy91244) [reply ajax/libs/big-integer/package.json]: @extend1994 
Aw, you are right!
I have modified the commit, please confirm, thank you.

**Pull request merged**